composer.json

```diff
@@ -8,7 +8,7 @@
     "prefer-stable": true,
     "require": {
         "php": "~5.5",
-        "doctrine/dbal": "2.3.5",
+        "doctrine/dbal": ">=2.3.5",
```

Comment:
I've tested it with <code>Doctrine DBAL</code> versions:
 * 2.3.5
 * 2.4.1
 * 2.5.1